ios/FluentUI/Shimmer/ShimmerView.swift

```diff
@@ -227,7 +269,7 @@ open class ShimmerView: UIView {
 		let dark = UIColor.black.cgColor
```

Comment:
Not part of your change, but I think it'd be nicer to have these light and dark colors in the constants too.

Reply:
Moved the dark color to the constant. Light color unfortunately cannot be part of constant because it depends on shimmerAlpha and is created dynamically.